components/tidb_query/src/expr_util/rand.rs

```diff
@@ -39,6 +39,7 @@ mod tests {
     #[allow(clippy::float_cmp)]
     fn test_rand_new() {
         let mut rng1 = MySQLRng::new();
+        std::thread::sleep(std::time::Duration::from_millis(100));
```

Comment:
Can we sleep a more short duration to avoid make unit tests slow?

Reply:
Unit tests are run in parallel and I think this is fine. Small throttles are not stable in our CI. Some coprocessor tests even sleep in 300ms unit. @lonng